playbooks/roles/tor-bridge/templates/torrc.j2

```diff
@@ -2,6 +2,7 @@ SocksPort 0
 ORPort {{ tor_orport }}
 ExtORPort auto
 BridgeRelay 1
+PublishServerDescriptor 0
```

Comment:
Can you make the value of this setting be populated from a variable in the [defaults.yml](https://github.com/jlund/streisand/blob/677d255849b4167b3390f77b1425bc24e476829a/playbooks/roles/tor-bridge/defaults/main.yml) for the `tor-bridge` recipe, similar to `{{ tor_orport }}` above? 

Keeping the value as `0` is great as the default but it would be nice to add a comment explaining that if you would like to opt-in to contributing a bridge to the Tor network that you should use the value `bridge` instead of `0`. Please also briefly describe what "0" does to save folks a manual read :-) 

Thanks!